colin/checks/labels/vcs-ref.py

```diff
@@ -0,0 +1,14 @@
+from colin.checks.abstract.labels import LabelCheck
+
+
+class VcsRefLabelCheck(LabelCheck):
+
+    def __init__(self):
+        super().__init__(name="vcs-ref_label",
```

Comment:
This is set automatically, based on the our document. I don't know if it make sense to have it in Fedora.